tests/src/quickgui/testqgsquickutils.cpp

```diff
@@ -51,9 +55,27 @@ void TestQgsQuickUtils::dump_screen_info()
 
 void TestQgsQuickUtils::screenUnitsToMeters()
 {
+  QgsCoordinateReferenceSystem crs3857 = QgsCoordinateReferenceSystem::fromEpsgId( 3857 );
+  QVERIFY( crs3857.authid() == "EPSG:3857" );
+
   QgsCoordinateReferenceSystem crsGPS = QgsCoordinateReferenceSystem::fromEpsgId( 4326 );
   QVERIFY( crsGPS.authid() == "EPSG:4326" );
 
+  QgsPointXY pointXY = utils.pointXYFactory( 49.9, 16.3 );
```

Comment:
Please separate to new test `transformPoint`, it does not belongs to `TestQgsQuickUtils::screenUnitsToMeters()`